server/handlers/events_streamer.go

```diff
@@ -475,3 +475,52 @@ func closeAdapterConnections(localMeshAdaptersLock *sync.Mutex, localMeshAdapter
 
 	return map[string]*meshes.MeshClient{}
 }
+
+// swagger:route POST /api/events EventsAPI idUIEventHandler
+// Handle sending UI-generated events to the notification center through the server.
+// responses:
+// 200:
+
+func (h *Handler) UIEventHandler(w http.ResponseWriter, req *http.Request, prefObj *models.Preference, user *models.User, provider models.Provider) {
```

Comment:
Could you think about changing the function name to something more meaningful? Please take a moment to come up with a name that better reflects its purpose. 
Is there a way to make it even better?

Reply:
i hope `ClientEventHandler` is good enough